memdx/errors.go

```diff
@@ -15,6 +15,7 @@ var (
 	ErrCollectionsNotEnabled               = errors.New("collections not enabled")
 	ErrDocNotFound                         = errors.New("document not found")
 	ErrDocExists                           = errors.New("document already exists")
+	ErrDocTooBig                           = errors.New("document too big")
```

Comment:
The RFC has this with the name `ValueTooLarge`, is the deviation from the RFC here intentional?

Reply:
Good spot, I forgot that the RFC exists.